### ISSUE TITLE
Can't find a SQL file

### ISSUE THREAD

**Issue by yangshuairocks** [XAMPPRocky/tokei#420]
I `git clone`d the `tokei` repo and use tokei to report the code info for this repo. There is a `.sql` file in this repo and it seems `tokei` can't find it, even with the parameter `--no-ignore`. Below is my testing.  Am I missing anything?

```
C:\my_projects\tokei>tokei
-------------------------------------------------------------------------------
 Language            Files        Lines         Code     Comments       Blanks
-------------------------------------------------------------------------------
 JSON                    1         2031         2031            0            0
 Markdown                5         2006         2006            0            0
 PowerShell              1           23           13            3            7
 Rust                   19         2949         2131          413          405
 Shell                   4          138          102            8           28
 TOML                    1           79           67            0           12
 YAML                    1           80           56           12           12
-------------------------------------------------------------------------------
 Total                  32         7306         6406          436          464
-------------------------------------------------------------------------------

C:\my_projects\tokei>tokei --no-ignore
-------------------------------------------------------------------------------
 Language            Files        Lines         Code     Comments       Blanks
-------------------------------------------------------------------------------
 JSON                    1         2031         2031            0            0
 Markdown                5         2006         2006            0            0
 PowerShell              1           23           13            3            7
 Rust                   19         2949         2131          413          405
 Shell                   4          138          102            8           28
 TOML                    1           79           67            0           12
 YAML                    1           80           56           12           12
-------------------------------------------------------------------------------
 Total                  32         7306         6406          436          464
-------------------------------------------------------------------------------

C:\my_projects\tokei>dir tests\data\sql.sql
 Volume in drive C has no label.
 Volume Serial Number is 442F-66DB

 Directory of C:\my_projects\tokei\tests\data

12/28/2019  01:48 AM               337 sql.sql
               1 File(s)            337 bytes
               0 Dir(s)  240,905,682,944 bytes free

C:\my_projects\tokei>
```

**Comment by XAMPPRocky** [comment]
@yangshuairocks Thank you for your issue! It seems that there's a regression and the `—no-ignore` flag is not triggering the correct behaviour. I'll investigate and try to release a new version to fix that soon.

**Comment by XAMPPRocky** [comment]
Okay, the issue was with having a custom ignore file. If you had a `.tokeignore` it would always ignore regardless of the option. I've published v10.1.1 that contains a fix for this. Thank you again.
